packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/covertGifToPng.ts

```diff
@@ -0,0 +1,30 @@
+import generateDataURL from './generateDataURL';
+import ImageEditInfo from '../types/ImageEditInfo';
+
+/**
+ * If the image is a gif, use canvas to convert it to a png.
```

Comment:
Add @internal

Reply:
There are other types of animated image extensions such as [WebP](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#webp_image), [AVIF](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#avif_image), [APNG](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#apng_animated_portable_network_graphics) in case we want to also support them.